### PR TITLE
SEOImages: User Story 1463880, Part 30

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Fixes #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw This PR is ready for review. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019032000040/Report/

**Comment by damabe** [comment]
Updated based on other PR reviews. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019032700053/Report/

File: docs/framework/wcf/feature-details/workflow-service-host-internals.md [hunk 1]
```diff
@@ -7,36 +7,36 @@ ms.assetid: af44596f-bf6a-4149-9f04-08d8e8f45250
 <xref:System.ServiceModel.WorkflowServiceHost> provides a host for workflow services. It is responsible for listening for incoming messages and routing them to the appropriate workflow service instance, it controls unloading and persisting of idle workflows, and more. This topic describes how WorkflowServiceHost processes incoming messages.  
   
 ## WorkflowServiceHost Overview  
- The <xref:System.ServiceModel.WorkflowServiceHost> class is used to host workflow services. It listens for incoming messages and routes them to the appropriate service instance, creating new instances or loading existing instances from durable storage as needed.  The following diagram illustrates on a high level how <xref:System.ServiceModel.WorkflowServiceHost> works.  
+ The <xref:System.ServiceModel.WorkflowServiceHost> class is used to host workflow services. It listens for incoming messages and routes them to the appropriate service instance, creating new instances or loading existing instances from durable storage as needed.  The following diagram illustrates on a high level how <xref:System.ServiceModel.WorkflowServiceHost> works:
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/workflow-service-host-internals.md]:
```suggestion

The <xref:System.ServiceModel.WorkflowServiceHost> class is used to host workflow services. It listens for incoming messages and routes them to the appropriate service instance, creating new instances or loading existing instances from durable storage as needed. The following diagram illustrates on a high level how <xref:System.ServiceModel.WorkflowServiceHost> works: 
```

File: docs/framework/wcf/feature-details/workflow-service-host-internals.md [hunk 2]
```diff
@@ -7,36 +7,36 @@ ms.assetid: af44596f-bf6a-4149-9f04-08d8e8f45250
...
   
 ## WorkflowServiceHost Overview  
- The <xref:System.ServiceModel.WorkflowServiceHost> class is used to host workflow services. It listens for incoming messages and routes them to the appropriate service instance, creating new instances or loading existing instances from durable storage as needed.  The following diagram illustrates on a high level how <xref:System.ServiceModel.WorkflowServiceHost> works.  
+ The <xref:System.ServiceModel.WorkflowServiceHost> class is used to host workflow services. It listens for incoming messages and routes them to the appropriate service instance, creating new instances or loading existing instances from durable storage as needed.  The following diagram illustrates on a high level how <xref:System.ServiceModel.WorkflowServiceHost> works: 
   
- ![WorkflowServiceHost Overview](../../../../docs/framework/wcf/feature-details/media/wfshhighlevel.gif "WFSHHighLevel")  
+ ![Diagram that shows a Workflow Service Host overview.](./media/workflow-service-host-internals/workflow-service-host-high-level-overview.gif)
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/workflow-service-host-internals.md]:
```suggestion
 ![Diagram that shows an overview of the Workflow Service Host.](./media/workflow-service-host-internals/workflow-service-host-high-level-overview.gif)  
```

File: docs/framework/wcf/feature-details/workflow-service-host-internals.md [hunk 3]
```diff
@@ -7,36 +7,36 @@ ms.assetid: af44596f-bf6a-4149-9f04-08d8e8f45250
...
+ ![Diagram that shows the flow when WorkflowServiceHost.Open is called.](./media/workflow-service-host-internals/workflow-service-host-open.gif)  
   
  The workflow is loaded from XAML and the activity tree is created. <xref:System.ServiceModel.WorkflowServiceHost> walks the activity tree and creates the service description. Configuration is applied to the host. Finally the host begins to listen for incoming messages.  
   
- The following illustration shows what the <xref:System.ServiceModel.WorkflowServiceHost> does when it receives a message bound for a Receive activity that has CanCreateInstance set to `true`.  
+ The following illustration shows what the <xref:System.ServiceModel.WorkflowServiceHost> does when it receives a message bound for a Receive activity that has CanCreateInstance set to `true`:  
   
- ![Workflow Service Host Receives a message](../../../../docs/framework/wcf/feature-details/media/wfhreceivemessagecci.gif "WFHReceiveMessageCCI")  
+ ![Image showing when WFS Host receives a message and CanCreateInstance is true.](./media/workflow-service-host-internals/workflow-service-host-receive-message-cancreateinstance.gif)
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/workflow-service-host-internals.md]:
```suggestion
 ![Decision tree used by the WFS Host when it receives a message and CanCreateInstance is true.](./media/workflow-service-host-internals/workflow-service-host-receive-message-cancreateinstance.gif)  
```

File: docs/framework/wcf/feature-details/workflow-service-host-internals.md [hunk 4]
```diff
@@ -7,36 +7,36 @@ ms.assetid: af44596f-bf6a-4149-9f04-08d8e8f45250
...
+ ![Image showing when WFS Host receives a message and CanCreateInstance is true.](./media/workflow-service-host-internals/workflow-service-host-receive-message-cancreateinstance.gif)  
   
  The message arrives and is processed by the WCF channel stack. Throttles are checked and correlation queries are executed. If the message is bound for an existing instance the message is delivered. If a new instance needs to be created, the Receive activity’s CanCreateInstance property is checked. If it is set to true, a new instance is created and the message is delivered.  
   
  The following illustration shows what the <xref:System.ServiceModel.WorkflowServiceHost> does when it receives a message bound for a Receive activity that has CanCreateInstance set to false.  
   
- ![WorkflowServiceHost receives a message](../../../../docs/framework/wcf/feature-details/media/wfshreceivemessage.gif "WFSHReceiveMessage")  
+ ![Image showing when WFS Host receives a message and CanCreateInstance is false.](./media/workflow-service-host-internals/workflow-service-host-receive-message.gif)
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/workflow-service-host-internals.md]:
```suggestion
 ![Decision tree used by the WFS Host when it receives a message and CanCreateInstance is false.](./media/workflow-service-host-internals/workflow-service-host-receive-message.gif)  
```

File: docs/framework/wpf/app-development/navigation-topologies-overview.md [hunk 5]
```diff
@@ -40,7 +40,7 @@ ms.assetid: 5d5ee837-629a-4933-869a-186dc22ac43d
   
  This topic covers three common navigation topologies: *fixed linear*, *fixed hierarchical*, and *dynamically generated*. Each navigation topology is demonstrated with a sample that has a [!INCLUDE[TLA2#tla_ui](../../../../includes/tla2sharptla-ui-md.md)] like the one that is shown in the following figure:  
   
- ![Task pages with data items](./media/navigationtopologyfigure6.png "NavigationTopologyFigure6")  
+ ![Screenshot that shows task pages with data items.](./media/navigation-topologies-overview/navigation-topology-data-items.png)
```
Comment (mairaw) [inline docs/framework/wpf/app-development/navigation-topologies-overview.md]:
```suggestion
 ![Task pages with data items and navigation buttons.](./media/navigation-topologies-overview/navigation-topology-data-items.png)  
```

File: docs/framework/wpf/controls/wpf-content-model.md [hunk 6]
```diff
@@ -32,9 +32,9 @@ ms.assetid: 214da5ef-547a-4cf8-9b07-4aa8a0e52cdd
 |<xref:System.Windows.Controls.ItemsControl>|A collection of arbitrary objects.|  
 |<xref:System.Windows.Controls.HeaderedItemsControl>|A header and a collection of items, all of which are arbitrary objects.|  
   
- Controls that inherit from these classes can contain the same type of content and treat the content in the same way. The following illustration shows one control from each content model that contains an image and some text.  
+ Controls that inherit from these classes can contain the same type of content and treat the content in the same way. The following illustration shows one control from each content model that contains an image and some text:  
   
- ![Button, GroupBox, Listbax, TreeViewItem](./media/controlcontentmodelimagetextinto.PNG "ControlContentModelImageTextInto")  
+ ![Screenshot that shows a control from each content model.](./media/wpf-content-model/control-content-model-image-text.png)
```
Comment (mairaw) [inline docs/framework/wpf/controls/wpf-content-model.md]:
```suggestion
 ![Screenshot that shows four different controls, one from each content model.](./media/wpf-content-model/control-content-model-image-text.png)  
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @damabe. Left you some comments before merging.

**Comment by damabe** [comment]
@mairaw Changes have been made. New CATS: https://contentqacats.azurewebsites.net/CheckResults/2019040200034/Report/

**Review by mairaw — APPROVED** [review]

**Pull request merged**